package.json

```diff
@@ -6,8 +6,10 @@
     "babel-generator": "^6.19.0",
     "babel-plugin-istanbul": "^4.1.3",
     "babel-plugin-module-alias": "^1.2.0",
+    "babel-plugin-transform-assertion-comments": "^0.3.1",
     "babel-plugin-transform-async-to-generator": "^6.8.0",
     "babel-plugin-transform-function-bind": "^6.3.13",
+    "babel-plugin-transform-metamagical-comments": "^0.14.0",
     "babel-plugin-transform-object-rest-spread": "^6.3.13",
     "babel-polyfill": "^6.13.0",
     "babel-preset-es2015": "^6.3.13",
```

Comment:
Ah, this is why your local tests error. These modules are symlinked from a git repo instead of installed from the npm repository. You need to [initialise the git submodules and run `make tools` instead](http://folktale.origamitower.com/docs/v2.0.0/contributing/setup/#cloning-and-initialising-the-repository) (yeah, it's… not great).

```shell
$ git submodule init
$ git submodule update
$ npm install
$ make tools
```